.github/workflows/build-with-bal-test-native.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up GraalVM
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '21.0.1'
```

Comment:
We can delete this work flow file. We already have the centralised workflow: https://github.com/ballerina-platform/module-ballerinax-aws.dynamodb/blob/main/.github/workflows/build-with-bal-test-graalvm.yml